app/assets/javascripts/application.js

```diff
@@ -1,3 +0,0 @@
-//= require_tree .
-//= require govuk_publishing_components/dependencies
-//= require govuk_publishing_components/all_components
```

Comment:
Did you mean for this file to be empty? and if so, do we need it?

Reply:
Fair question – I was thinking we might want to add some JS here in the future, but given nobody is going to run Search Admin on non-ES6 browsers I'll remove it entirely!